Decorator/TextDecorator/MessageDecorator.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace Decorator
+{
+    public class MessageDecorator : IMessage
```

Comment:
This base decorator like an interface. It's needed only for creating concrete decorators.

Reply:
done

Reply:
Now it's abstract again